case-lib/lib.sh

```diff
@@ -222,12 +224,12 @@ storage_checks()
       # non-zero exit status when taking too long.
       time timeout -s CONT "$max_sync" sudo sync || return $?
       # Spend a few seconds to test and show the current write speed
-      timeout -s CONT 5 dd if=/dev/zero of=~/HD_TEST_DELETE_ME bs=1M count="$megas" conv=fsync ||
+      timeout -s CONT 5 dd if=/dev/zero of=$WRITE_SPEED_CHECK_DUMMY_FILE bs=1M count="$megas" conv=fsync ||
           return $?
       time timeout -s CONT "$max_sync" sudo sync
     ) || return $?
 
-    rm ~/HD_TEST_DELETE_ME
+    rm $WRITE_SPEED_CHECK_DUMMY_FILE
```

Comment:
Also not sure why this write check was added unconditionally, it was meant to debug some issues with mass storage but I don't think it needs to be a key feature that is run in all cases on all platforms, it should be opt-in. @marc-hb this is likely for you to comment on.